### ISSUE TITLE
Model will not train

### ISSUE THREAD

**Issue by hackelle** [slwang9353/MobileFormer#7]
When the code is run in google colab, the Validation accuracy will not improve beyond around 0.11 on cifar-10 when running the search and is often even below 0.1

**Comment by jyxgithubalpha** [comment]
I got the same result too. Have you figured out the reason?

**Comment by hackelle** [comment]
Unfortunately I did not. I switched to using [this implementation](https://github.com/ACheun9/Pytorch-implementation-of-Mobile-Former)
